resources/sdk/clisdkclient/extensions/utils/general.go

```diff
@@ -305,3 +309,19 @@ func MilliSecondsToNanoSeconds(milliSeconds int64) time.Duration {
 func SecondsToNanoSeconds(seconds int) time.Duration {
 	return MilliSecondsToNanoSeconds(int64(seconds)) * 1000
 }
+
+func convertToJSON(data string) string {
+	if strings.EqualFold("json", data_format.InputFormat) && isJSON(data) {
+		return data
+	}
+	result, err := yaml.YAMLToJSON([]byte(data))
+	if err != nil {
+		logger.Fatal(fmt.Printf("err: %v\n", err))
```

Comment:
Logging the result of fmt.Printf() would result in some weird output. Best to do something like "logger.Fatalf("Error converting YAML to JSON: %v\n", err)"